tests/form_test.go

```diff
@@ -0,0 +1,35 @@
+package tests
+
+import (
+	"strings"
+	"testing"
+
+	testutils "github.com/HexmosTech/lama2/tests/utils"
+)
+
+func TestFormPositive(t *testing.T) {
```

Comment:
Have we verified none of the `MULTIPART` cases are not breaking?

Just verify how postman deals with a multipart without file attachment once. I think we should verify that case works (if a testcase not there, we can add that as well).